lib/v2/behance/user.js

```diff
@@ -32,7 +38,12 @@ module.exports = async (ctx) => {
             const articleHtml = response2.data;
             const $2 = cheerio.load(articleHtml);
 
-            const content = $2('div.project-styles').html();
+            $2('.ImageElement-root-kir').remove();
+            $2('.embed-dimensions').remove();
+            const content = $2('div.project-styles')
+                .html()
+                .replace(/<.?script>/g, '')
+                .replace(/<script*.+js-lightbox-slide-content">/g, '');
```

Comment:
Why not just use cheerio to remove them?

Reply:
Because some images is not showing up, but I found there are just inside script node.
```
<script type="text/html" class="js-lightbox-slide-content">
          <div class="project-lightbox-image-container">
            <img src="https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/4744d3134074501.62012a95d5791.jpg" srcset="https://mir-s3-cdn-cf.behance.net/project_modules/disp/4744d3134074501.62012a95d5791.jpg 600w,https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/4744d3134074501.62012a95d5791.jpg 1200w,https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/4744d3134074501.62012a95d5791.jpg 1400w," sizes="(max-width: 1400px) 100vw, 1400px">
            <div class="project-item-lightbox__actions lightbox-extra">
              <div class="project-item-lightbox__action js-grid-item-lightbox-add-to-collection" data-id="119587313"></div>
            </div>
          </div>
          <div class="project-item-lightbox__caption js-lightbox-caption" data-id="119587313"></div>
        </script>
```

Reply:
Try this?

```js
$2('div.project-styles script.js-lightbox-slide-content').each(() => $(this).replaceWith($(this).html()));
```

Reply:
I don't know what's wrong
```
            const $2 = cheerio.load(articleHtml);
            $2('.ImageElement-root-kir').remove();
            $2('.embed-dimensions').remove();
            //$2('script.js-lightbox-slide-content').each(()=> $2(this).unwrap());
            //$2('div.project-styles script.js-lightbox-slide-content').each(() => $2(this).replaceWith($2(this).html()));
            const content = $2('div.project-styles')
                .html()
```
None of these works.

Reply:
```js
$2('script.js-lightbox-slide-content').each((_, elem) => $2(elem).replaceWith($2(elem).html()));
```